ansible/asqatasun/templates/xvfb.j2

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
I had enhanced the `xvfb` script, could you sync this one with the original one ([`xvfb` script in docker/single-container/](https://github.com/Asqatasun/Asqatasun/blob/develop/docker/single-container/xvfb)) ?